spec/validators/program_validator_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rspec'
+
+describe ProgramValidator do
+  describe 'when validating for load_csv context' do
```

Comment:
Fix up wording as suggested in `edu_program_validator_spec.rb`

Reply:
changed